docs/runtime/nodejs-metrics.md

```diff
@@ -0,0 +1,310 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Node.js Runtime
+--->
+
+# Semantic Conventions for Node.js Runtime Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for Node.js Runtime metrics in OpenTelemetry.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Experimental](#experimental)
+  - [Metric: `nodejs.eventloop.delay.min`](#metric-nodejseventloopdelaymin)
+  - [Metric: `nodejs.eventloop.delay.max`](#metric-nodejseventloopdelaymax)
+  - [Metric: `nodejs.eventloop.delay.mean`](#metric-nodejseventloopdelaymean)
+  - [Metric: `nodejs.eventloop.delay.stddev`](#metric-nodejseventloopdelaystddev)
+  - [Metric: `nodejs.eventloop.delay.p50`](#metric-nodejseventloopdelayp50)
+  - [Metric: `nodejs.eventloop.delay.p90`](#metric-nodejseventloopdelayp90)
+  - [Metric: `nodejs.eventloop.delay.p99`](#metric-nodejseventloopdelayp99)
+  - [Metric: `nodejs.eventloop.utilization`](#metric-nodejseventlooputilization)
+
+<!-- tocstop -->
+
+## Experimental
+
+**Status**: [Experimental][DocumentStatus]
+
+**Description:** Experimental Node.js Runtime metrics captured under `nodejs`.
+
+### Metric: `nodejs.eventloop.delay.min`
+
+This metric is [recommended][MetricRecommended].
```

Comment:
Could you give a brief description of how this metric is recorded, like it's done for Java:

> This metric is obtained from [`MemoryPoolMXBean#getUsage()`](https://docs.oracle.com/javase/8/docs/api/java/lang/management/MemoryPoolMXBean.html#getUsage--).

Or Go:

> [1]: Computed from `/sched/goroutines:goroutines`.